vision/realsense/camera_driver.py

```diff
@@ -0,0 +1,56 @@
+"""
+This program was written to test/be an example for the camera child classes
```

Comment:
I believe I saw that you added the readme to the other pull request. If so lets delete this file and add its contents to that readme.